homeassistant/components/zwave_js/api.py

```diff
@@ -2498,3 +2506,77 @@ def _handle_device_added(device: dr.DeviceEntry) -> None:
         )
     ]
     await driver.async_hard_reset()
+
+
+@websocket_api.websocket_command(
+    {
+        vol.Required(TYPE): "zwave_js/node_capabilities",
+        vol.Required(DEVICE_ID): str,
+    }
+)
+@websocket_api.async_response
+@async_handle_failed_command
+@async_get_node
+async def websocket_node_capabilities(
+    hass: HomeAssistant,
+    connection: ActiveConnection,
+    msg: dict[str, Any],
+    node: Node,
+) -> None:
+    """Get node endpoints with their support command classes."""
+
+    connection.send_result(
+        msg[ID],
+        {
+            idx: [command_class.to_dict() for command_class in endpoint.command_classes]
```

Comment:
I think we said that the frontend wanted snake_case keys. We need to adjust that here.